.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2
+jobs:
+  build-and-test:
+    docker:
+      - image: uxpin2/node8:1.0
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+    working_directory: ~/project/
+    steps:
+      - checkout
+      - run:
```

Comment:
Let's remove "Login to NPM". Boilerplate is a **public** repository. We shouldn't install anything that is private.